tldw_Server_API/app/core/Metrics/traces.py

```
@@ -404,10 +408,7 @@ async def async_wrapper(*args, **kwargs):
 
                         return result
 
-                    except Exception as e:
-                        if span:
-                            span.record_exception(e)
-                            span.set_status(Status(StatusCode.ERROR, str(e)))
+                    except Exception:
                         raise
```

Ruff (0.15.9)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)